src/otp/otp.service.ts

```diff
@@ -54,14 +55,10 @@ export class OtpService {
     //MOCKING RESPONSE
     if (token === process.env.MOCKING_OTP_TOKEN && pin === process.env.MOCKING_OTP) {
       return {
-        status: "success",
-        message: "Code is correct."
-      }
-    } else {
-      return {
-        status: "fail",
-        message: "Code is incorrect."
-      }
+        status: 'success',
+        message: 'Code is correct.',
+      };
     }
+    throw new HttpException('Failed Verifying OTP', HttpStatus.BAD_REQUEST)
```

Comment:
change @ApiUnauthorizedResponse({ description: 'Failed Verifying OTP' }) in otp.controller to @ApiBadRequestResponse